Examples/Algorithms/Digitization/src/MeasurementCreation.cpp

```diff
@@ -30,6 +30,7 @@ ActsExamples::VariableBoundMeasurementProxy ActsExamples::createMeasurement(
   return Acts::visit_measurement(
       dParams.indices.size(), [&](auto dim) -> VariableBoundMeasurementProxy {
         auto [indices, par, cov] = measurementConstituents<dim>(dParams);
-        return container.emplaceMeasurement<dim>(geometryId, indices, par, cov);
+        return VariableBoundMeasurementProxy{
+            container.emplaceMeasurement<dim>(geometryId, indices, par, cov)};
```

Comment:
That looks a bit odd to me. I suspect that we cannot convert from `FixedBoundMeasurementProxy` anymore? If we want to keep that it might be cleaner to do something like

```cpp
return VariableBoundMeasurementProxy(Acts::visit_measurement(
      dParams.indices.size(), [&](auto dim) {
        auto [indices, par, cov] = measurementConstituents<dim>(dParams);
        return container.emplaceMeasurement<dim>(geometryId, indices, par, cov);
        return container.emplaceMeasurement<dim>(geometryId, indices, par, cov)};
      }));
```
?

Reply:
It seems, that we need an explicit return type for the lambda:

```
error: inconsistent deduction for auto return type: ‘ActsExamples::FixedMeasurementProxy<6, 5, false>’ and then ‘ActsExamples::FixedMeasurementProxy<6, 6, false>’
```